_posts/2023-10-08-amazon_bedrock_part1.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: postv2
+font: serif
+title: "Amazon Bedrock Anatomy"
+description: "Part1: What's Amazon Bedrock and what it does?"
+date: 2023-10-08
+author: Nadia Reyhani
+tags: [Machine Learning, Amazon Web Services, ML, Generative AI, Amazon Bedrock, AWS]
+---
+
+## Introduction
+
+Did you also catch the thrilling announcement? "Amazon Bedrock" is now officially accessible to all users. While a few incredible features of this service are still in the "Preview" stage, it's sufficiently prepared to empower us with exciting capabilities for effortlessly creating and launching Generative AI applications.
```

Comment:
also? in addition to what? 
I'd suggest "Did you catch the thrilling announcement?"